lib/minfraud/components/credit_card.rb

```diff
@@ -62,17 +62,28 @@ class CreditCard < Base
       # @return [String, nil]
       attr_accessor :cvv_result
 
+      # Whether the outcome of 3-D Secure verification (e.g. Safekey,
+      # SecureCode, Verified by Visa) was successful. `true` if customer
+      # verification was successful, or `false` if the customer failed
```

Comment:
I believe markdown does not work in these comments. `+false+` would. (Actually I think yard can be made to support markdown but I don't think ours is set up to do that? I'm not sure. Either way it is not working AFAICT).